src/pages/PlanPage/subRoutes/CurrentOrgPlan/BillingDetails/PaymentCard/CreditCardForm.tsx

```diff
@@ -17,21 +25,21 @@ function CreditCardForm({ closeForm, provider, owner }) {
     owner,
   })
 
-  function submit(e) {
+  function submit(e: any) {
```

Comment:
kind of trivial but we could do `e: React.FormEvent`

Reply:
We could possibly file this as a followup ticket to get this feature change out and then also do an investigation to change any other forms not using react-hook-form along with this one at the same time.

Reply:
Quickly skimming through the codebase, this looks to be the only form that currently does not use `react-hook-form` to manage the state for the various elements of the form.

Reply:
Great! Created https://github.com/codecov/engineering-team/issues/1884 to track that work